dali/test/python/test_dali_cpu_only.py

```diff
@@ -30,10 +28,9 @@
 from nose_utils import assert_raises
 import os
 import glob
-import tempfile
-import json
 from collections.abc import Iterable
 from nose.plugins.attrib import attr
+from test_dali_cpu_only_utils import *
```

Comment:
nitpick: maybe enumerate what you import here

Reply:
done